test/llvm/generate-nsw-for-signed/nsw-generation.chpl

```diff
@@ -23,8 +23,6 @@ proc test()
   signedMul   = aSigned * bSigned;
   // CHECK: sub nsw i64
   signedSub   = aSigned - bSigned;
-  // CHECK: shl nsw i64
-  signedShl   = aSigned << bSigned;
```

Comment:
I saw some comment from @mppf about LLVM. I am not sure but maybe this test can be skipif'ed or something (when there is boundsCheck)

Reply:
The test is effectively pointless at this moment because the Chapel bitshift routines circumvent the code generated by LLVM for signed shifts.